assets/runtime/env-defaults

```diff
@@ -414,6 +414,7 @@ GITLAB_PAGES_HTTPS=${GITLAB_PAGES_HTTPS:-false}
 GITLAB_PAGES_EXTERNAL_HTTP=${GITLAB_PAGES_EXTERNAL_HTTP:-}
 GITLAB_PAGES_EXTERNAL_HTTPS=${GITLAB_PAGES_EXTERNAL_HTTPS:-}
 GITLAB_PAGES_ACCESS_CONTROL=${GITLAB_PAGES_ACCESS_CONTROL:-false}
+GITLAB_PAGES_NGINX_DISABLE=${GITLAB_PAGES_NGINX_DISABLE:-false}
```

Comment:
I don't like the naming of this variable, because I think the for the user it's not directly clear what this variable can do. I think more clear way would be `NGINX_GITLAB_PAGES_PROXY` or `GITLAB_PAGES_NGINX_PROXY`

Reply:
Changes Done.

Variable renamed: `GITLAB_PAGES_NGINX_PROXY` changed, default, from `false` to `true` and updated all corresponding functions so that the variable makes sense.

`GITLAB_PAGES_NGINX_PROXY` Default: `true`

When set to `false` it will disable the proxy, README has been updated

@solidnerd Request Check